ProjectEuler/Problem18/p018.py

```diff
@@ -0,0 +1,23 @@
+import time
+import numpy as np
+
+t_0 = time.time()
+
+pt = np.loadtxt('padded_tri.txt')
+
+i = 0
+while i < 15:
+	nonzero = np.count_nonzero(pt[i]) - 1
```

Comment:
I dunno why that tab was inconsistent; I'll change it and update it in the next pull request.